java/fury-core/src/main/java/org/apache/fury/util/Platform.java

```diff
@@ -143,6 +143,40 @@ public static long objectFieldOffset(Field f) {
     return UNSAFE.objectFieldOffset(f);
   }
 
+  /**
+   * Gets the offset of the only field in the class matching the required type.
+   *
+   * @param clazz the class in which the field should be declared
+   * @param fieldType the required type of the field
+   * @return offset of the field
+   * @throws IllegalStateException if there are multiple fields of the required type
+   * @throws IllegalArgumentException if there are no fields of the required type
+   */
+  public static long objectFieldOffset(Class<?> clazz, Class<?> fieldType) {
```

Comment:
Could we move this method to `ReflectionUtils`? Platform is a more internal API which expose jdk internals. We try to make it simple and leave more logic under ReflectioUtils

Reply:
done